csrc/scheduler/normalization_inner_outer_tma_ws.cpp

```diff
@@ -382,10 +303,7 @@ void scheduleOuterReduction(
     }
 
     outer_reduction_tv->axis(axisID--)->parallelize(ParallelType::BIDy);
-
-    auto outer_reference_tv =
-        reduction_scheduler_utils::sortAndRFactor(outer_reduction_tv);
-    outer_reference_tvs.emplace_back(outer_reference_tv);
+    outer_reference_tvs.emplace_back(outer_reduction_tv);
```

Comment:
The final outer reduction is a serial reduction of 132 (sm count) elements, no need to use `rFactor`

Reply:
Let's put this in as code comment instead.